src/lucky/renderable.cr

```diff
@@ -247,6 +247,11 @@ module Lucky::Renderable
     head(status.value)
   end
 
+  # allows json-compatible string to be returned directly
+  def json(body : String, status : Int32? = nil) : Lucky::TextResponse
+    send_text_response(body, "application/json", status)
```

Comment:
Should we still `JSON.parse` the body, here?

My only concern would be that this allows me to do `json "HAHA THIS ISN'T JSON"`, and now my app is silently returning invalid content for the content type `application/json`.